### ISSUE TITLE
[Boss] Auriaya vol. II

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#2807]
Some things that needed further tests/research:

1) Rip Flesh
http://www.wow-mania.com/armory/?spell=64375

Stacking Bleed effect used by Auriaya's Sanctum Sentries http://www.wow-mania.com/armory/?npc=34014

This effect has a 10second cooldown per cat to be cast, which currently only works on the first attempt of the fight. If you reset or wipe, the cooldown is gone due to the cats reseting wrongly, and on next attempt they will spam the bleed non stop, giving you 30stacks in a few seconds.

2) Fear Ward and Terrifying Screech
http://www.wow-mania.com/armory/?spell=64386
http://www.wow-mania.com/armory/?spell=6346
- Fear Ward does not get consumed by Terrifying Screech, (but will protect you from it)
- Terrifying Screech seems to be cast double (or multiple) within a short time range after the spell goes off
- Terrifying Screech has 5sec duration when it should be 3sec based on database and retail videos

**Comment by squalllh1989** [comment]
About the Fear Ward, last time at General Zarithrian, my fear ward protect me from being feared, and doesn't consume also.

**Comment by Dranah** [comment]
Hmm that part could be a general issue.
The complex thing about synergy between FW and Terrify is that the Terrifying Screech is cast more than just once atm, so the only reason FW currently works against it is the fact it doesn't get consumed.
I found this by having a fear ward nearly expiring when Auriaya cast Terrifying Screech, and that raid member getting feared 2-3 seconds later than others do (when FW expired), after the "original fear" did not affect him due to fear ward active.
Basically if the consumption of fear ward gets fixed, the spell will be consumed by the first fear and then the person will be feared by the second, as if the ward didn't work.
Keader have already reproduced this on PTR and is working on it.

**Comment by Keader** [comment]
Intimidating roar working fine in ptr.
Both issues fixed @Dranah

**Comment by Dranah** [comment]
UPDATE:
- Fixed issue with cats forgetting the cooldown on their bleed when they reset
- Fixed issue of Fear Ward not getting consumed by Terrifying Screech and Zarithrian's Intimidating Roar
- Terrifying Screech is not being cast multiple times anymore and now has proper 3sec duration down from 5sec.
